.github/workflows/release.yaml

```diff
@@ -148,7 +148,7 @@ jobs:
         uses: orhun/git-cliff-action@v3
         id: git-cliff
         with:
-          config: pyproject.toml
+          config: .cliff.toml
```

Comment:
The alternative solution (instead of having a Git Cliff TOML) is to delete the TOML and use whatever default settings the changelog generator runs with. This would work and avoid the need to have a new configuration file in the root of the repo.

Although we have a mix of standard commit messages and conventional commits in our mainline history, so I do see some benefit now in being "conventional commit"-aware when generating release notes so that the releases are more readable and the conventional commit prefixes are stripped.

https://github.com/fulcrumgenomics/fgbio/commits/main/